src/components/side-panel/layer-panel/vis-config-slider.js

```diff
@@ -50,31 +85,123 @@ export default function VisConfigSliderFactory(RangeSlider) {
     disabled,
     onChange,
     inputTheme
-  }) => (
-    <SidePanelSection disabled={Boolean(disabled)}>
-      {label ? (
-        <PanelLabel>
-          {typeof label === 'string' ? (
-            <FormattedMessage id={label} />
-          ) : typeof label === 'function' ? (
-            <FormattedMessage id={label(config)} />
+  }) => {
+    const [custom, setCustom] = useState(false);
+    const [values, setValues] = useState({
```

Comment:
instead of create a state here for value, value, value1, I would defer to have the actual input component to be stateful, 
for example, you can create a lazy input component, that will only call `onChange` when key is pressed and onBlue
```js
  const LazyInput = ({value, onChange, name}) => {
    const [stateValue, setValue] = useState(value);
    const inputRef = useRef(null);
    useEffect(() => {
      setValue(value);
    }, [value]);

    const onKeyDown = useCallback(
      e => {
        switch (e.keyCode) {
          case KeyEvent.DOM_VK_ENTER:
          case KeyEvent.DOM_VK_RETURN:
            onChange(stateValue);
            if (inputRef !== null) {
              // @ts-ignore
              inputRef?.current.blur();
            }
            break;
          default:
            break;
        }
      },
      [onChange, stateValue]
    );

    const _onChange = useCallback(e => setValue(e.target.value), [setValue]);
    const onBlur = useCallback(() => onChange(name, stateValue), [onChange, name, stateValue]);

    return (
      <RangeInput
        type="number"
        ref={inputRef}
        value={stateValue}
        onChange={_onChange}
        onBlur={onBlur}
        onKeyDown={onKeyDown}
        id={name}
      />
    );
  };
```